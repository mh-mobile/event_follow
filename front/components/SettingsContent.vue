<template lang="pug">
  div
    SettingsAccountDeletion
</template>

<script lang="ts">
import SettingsAccountDeletion from "@/components/SettingsAccountDeletion.vue"
import { defineComponent } from "@nuxtjs/composition-api"
import "firebase/auth"

export default defineComponent({
  components: {
    SettingsAccountDeletion
  }
})
</script>
