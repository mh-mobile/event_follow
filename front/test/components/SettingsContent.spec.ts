import { createLocalVue, mount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import SettingsContent from "@/components/SettingsContent.vue"

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
describe("SettingsContent.vue", () => {
  it("renders html correctly", () => {
    const wrapper = mount(SettingsContent, {
      localVue
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
