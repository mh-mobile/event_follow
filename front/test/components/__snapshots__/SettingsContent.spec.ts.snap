// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsContent.vue renders html correctly 1`] = `
"<div>
  <div>
    <div class=\\"account-deletion-button\\">退会</div>
    <!---->
  </div>
</div>"
`;
